sonar-flex-plugin/src/test/java/org/sonar/plugins/flex/FlexRulesDefinitionTest.java

```diff
@@ -20,22 +20,27 @@
 package org.sonar.plugins.flex;
 
 import org.junit.Test;
+import org.sonar.api.SonarRuntime;
```

Comment:
should not be part of this commit

Reply:
This change is brought by the latest version of the API that requires this additional SonarRuntime parameter to load the rules metadata (see `FlexRulesDefinition.define(Context context)`)

Reply:
Moved to the commit where versions are updated